cachi2/core/models/input.py

```diff
@@ -47,9 +47,11 @@ def show_error(error: "ErrorDict") -> str:
 
 
 # Supported package managers
-PackageManagerType = Literal["gomod", "npm", "pip", "yarn"]
+PackageManagerType = Literal["gomod", "npm", "pip"]
```

Comment:
We already had "yarn" in the list (prematurely?), but its presence had no effect on unit tests, so there's no reason to drop it from the list now to add it back in last commit, let's just leave the list as is.

Reply:
Right, but really, it **did** come in prematurely, and I'd like to see it in a commit where it makes sense.